tools/universe/package_builder.py

```diff
@@ -16,7 +16,7 @@
 logger = logging.getLogger(__name__)
 logging.basicConfig(level=logging.DEBUG, format="%(message)s")
 
-_jre_url = "https://downloads.mesosphere.com/java/server-jre-8u192-linux-x64.tar.gz"
+_jre_url = "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u212-b03/OpenJDK8U-jdk_x64_linux_hotspot_8u212b03.tar.gz"
```

Comment:
There are some pending questions about whether to download directly from openjdk's github or move to `downloads.mesosphere`. Let's wait on that decision and follow what happens in dcos-commons